src/ks-app/src/app/containers/iconography/icon-selection.component.ts

```diff
@@ -6,15 +6,15 @@
 
 import { Component } from '@angular/core';
 import { ClarityIcons } from '@clr/icons';
-import { ChartShapes } from '@clr/icons/shapes/chart-shapes';
-import { CommerceShapes } from '@clr/icons/shapes/commerce-shapes';
-import { CoreShapes } from '@clr/icons/shapes/core-shapes';
-import { EssentialShapes } from '@clr/icons/shapes/essential-shapes';
-import { MediaShapes } from '@clr/icons/shapes/media-shapes';
-import { SocialShapes } from '@clr/icons/shapes/social-shapes';
-import { TechnologyShapes } from '@clr/icons/shapes/technology-shapes';
-import { TextEditShapes } from '@clr/icons/shapes/text-edit-shapes';
-import { TravelShapes } from '@clr/icons/shapes/travel-shapes';
+import { ChartShapes } from '@clr/icons/shapes/chart';
```

Comment:
Can we just move these imports up to `main.ts` like in the dev app and do `AllShapes` instead?